docs/02.API-REFERENCE.md

```diff
@@ -1084,6 +1084,84 @@ jerry_get_global_object (void);
 
 Functions to check the type of an API value ([jerry_value_t](#jerry_value_t)).
 
+
+## jerry_value_instanceof
+
+**Summary**
+
+Returns whether the given `jerry_value_t` is an instance of the given prototype.
```

Comment:
Isn't there any difference between the es5.1 vs es2015 `instanceof` internal workings? Could that affect the api workings?

Reply:
It should not affect the api workings, but may affect the internal workings (e.g.: instance of classes).

Reply:
Classes are simple functions, the only specializations they have is a bytecode status flag, so I think it couldn't have any affect on it.